tests/console/snapper_create.pm

```diff
@@ -20,7 +20,13 @@ use utils;
 # that we got what we expect. See poo#25716
 sub get_last_snap_number {
     # get snapshot id column, tail before head to avoid SIGPIPE
-    my $snap_head         = script_output("snapper list | tail -n +1 | head -n1");
+    my $snap_head = script_output("snapper list | tail -n +1 | head -n1");
+    if ($snap_head =~ m/\[/) {
```

Comment:
Hmm, we are moving back and forth here. I've added code to surround output with `>>><<<` so it's easier to parse. And I also thought that we have disabled/stubbed btrfs maintenance jobs. So I would suggest at least to research root cause of the issue, especially knowing that we act on those issues and there should be no message like that on serial.

Reply:
maybe snapper triggers that message - it always appears in this test at this position.

Reply:
Can be, but still, I wonder why we do not disable btrfs maintenance here. @OleksandrOrlov, could you please point us to the code where we create symlinks to fake it? As for code, I would then use `#` as identifier that we have right line to parse, or re-run command if we get more than a single line, but with `[` it will break with any other scenario. I'm even surprised how stable this failure is.

Reply:
Sure, but then we don't need this condition we have for `[` symbol, as this thing will solve one particular failure and not other scenarios.